aosvc/install.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
 sudo cp aosvc /usr/local/bin
-sudp cp aosvc.service /etc/systemd/system
+sudo cp aosvc.service /etc/systemd/system
```

Comment:
I think `sudo` is not even needed as per https://github.com/ilyatbn/aws_alwayson/pull/4